pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -601,17 +613,16 @@ func getSpecEnv(envs []corev1.EnvVar, name string) string {
 	return ""
 }
 
-func getSmartRouterProtocol(cr *api.KieApp) string {
-	if cr.Status.Applied.Objects.SmartRouter != nil {
-		if len(cr.Status.Applied.Objects.SmartRouter.Protocol) == 0 && cr.Status.Applied.CommonConfig.DisableSsl {
-			return constants.HttpProtocol
-		} else if len(cr.Status.Applied.Objects.SmartRouter.Protocol) == 0 && !cr.Status.Applied.CommonConfig.DisableSsl {
-			return constants.HttpsProtocol
-		} else {
-			return cr.Status.Applied.Objects.SmartRouter.Protocol
-		}
```

Comment:
This is dropped completly? If this property is still there we should probably keep it here.